securing_apps/topics/oidc/oidc-generic.adoc

```diff
@@ -103,6 +103,15 @@ The device authorization endpoint is used to obtain a device code and a user cod
 
 For more details on how to invoke on this endpoint, see https://tools.ietf.org/html/rfc8628[OAuth 2.0 Device Authorization Grant specification].
 
+===== Backchannel Authentication Endpoint
+....
+/realms/{realm-name}/protocol/openid-connect/ext/ciba/auth
+....
+
+The backchannel authentication endpoint is used to obtain an auth_req_id that identifies the authentication request made by the client. It can only be invoked by confidential clients.
+
+For more details on how to invoke on this endpoint, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html[OpenID Connect Client Initiated Backchannel Authentication Flow specification].
```

Comment:
The CIBA documentation involves multiple places in the documentation, so it might be good to add some more "Cross-linking" so that administrator is able to easily refer to all the places with CIBA related stuff in our documentation.

For this last sentence, I suggest to add something like this `Also please refer to other places of {project_name} documentation like <Link to "Client Initiated Backchannel Authentication Grant" of this securing_apps guide> and <Client Initiated Backchannel Authentication Grant of server admin guide> .`

Reply:
I see, I would like to add such a cross-linking sentence.

Reply:
+1, Thanks!